docs/graphback-datasync.md

```diff
@@ -0,0 +1,230 @@
+---
+id: graphback-datasync
+title: Graphback Data Synchronization package
+sidebar_label: Setting up Data Synchronization using Graphback
+---
+
+The `@graphback/datasync` package consisting of the Data Synchronization Schema plugin and compatible data sources, provides out of the box Data Synchronization strategies for GraphQL clients with offline functionality e.g. [Offix](https://offix.dev). Currently this plugin **only** supports MongoDB data sources, with support for other kinds of data sources coming in a future release.
+
+## Motivation
+
+The raison d'être of the `@graphback/datasync` package is to provide additional out of the box functionality to offline-first GraphQL clients, mainly targeting [Offix](https://offix.dev). To that end, it has two main goals to fulfill:
```

Comment:
```suggestion
The`@graphback/datasync` package provide ability:

- Fetch data that was changed based on the client side timestamp
- Ensure data consistency using timestamp provided.
 
Graphback datasync extends [Offix](https://offix.dev) 
```

Reply:
Should be:

`The`@graphback/datasync` package provides the ability to:`

Reply:
I'd rather see more straightforward language than "raison d'être", like "purpose". You and I know what this means, but not everybody, and the less Googling of word meanings the better.

Reply:
Yep, agreed, just mistake in adding changes, wasn't really meant to be pushed :smile: